### PR TITLE
🚀 Deployment Optimizations

### PR DESCRIPTION
**Pull request by usamazansari**
Verify Deployment at [API](https://itinerary-assistant-api-ginq5m5mva-oa.a.run.app)

### CODE REVIEW

File: libs/api/database/neo4j/src/lib/models/configuration/configuration.model.ts [hunk 1]
```diff
@@ -1,60 +1,38 @@
...
 }
 
 const Neo4jConfigStub: INeo4jConfig = {
-  scheme: Neo4jScheme.NEO4J,
-  host: 'localhost',
-  port: 7687,
-  database: 'neo4j',
+  uri: 'neo4j://localhost:7687',
   username: 'neo4j',
   password: 'secret'
```
Comment (usama251993) [inline libs/api/database/neo4j/src/lib/models/configuration/configuration.model.ts:13]: Move these to `configuration.constants.ts`

File: libs/api/database/seed/src/lib/scripts/people.csv [hunk 2]
```diff
@@ -0,0 +1,49 @@
+"_id","_labels","apartment","city","coordinates","country","dateOfBirth","email","end","firstName","fullName","gender","id","landmark","lastName","locality","middleName","name","nickname","number","phone","plusCode","room","salutation","start","state","street","suburb","suffix","type","url","website","wing","zip","_start","_end","_type"
```
Comment (usama251993) [inline libs/api/database/seed/src/lib/scripts/people.csv:1]: 🔥 Remove sensitive information

File: libs/api/database/seed/src/lib/scripts/people.cypher [hunk 3]
```diff
@@ -0,0 +1,58 @@
+:begin
```
Comment (usama251993) [inline libs/api/database/seed/src/lib/scripts/people.cypher:1]: 🔥 Remove sensitive information

File: package-lock.json [hunk 4]
```diff
@@ -1,12 +1,12 @@
 {
```
Comment (usama251993) [inline package-lock.json:1]: 🙈 Remove `package-lock.json` from version control

**Review by usama251993 — CHANGES_REQUESTED** [review]
Review the comments

**Review by usama251993 — APPROVED** [review]
OK

File: package-lock.json (hunk 4 above)
Comment (usamazansari) [reply package-lock.json:1]: > 🙈 Remove `package-lock.json` from version control

Invalid

**Pull request merged**